### DISCUSSION TITLE
SSLCertVerificationError

### DISCUSSION THREAD

**Discussion by paolomariaschiratti** [Hyundai-Kia-Connect/hyundai_kia_connect_api#364] [Q&A]
I'm using the latest version of Hyundai_kia_connect_api with all the pkgs installed, but while executing monitor.py I receive this error:

 File "C:\Users\paoloSC\AppData\Local\Programs\Python\Python311\Lib\site-packages\requests\adapters.py", line 517, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /neoPix/bluelinky-stamps/master/hyundai-014d2225-8495-4735-812d-2616334fd15d.v2.json (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1002)')))

I've installed already the latest PY certificates using py -m pip install certifi.

Any idea of the reason of the failure?

**Comment by cdnninja** [comment]
Can you browse to the url and confirm the Certs are valid?  That may be a local network issue.

**Reply by paolomariaschiratti** [reply]
This is what I get from my browser, usign the URL

![image](https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/assets/140119554/e04577a2-0a5d-4cb0-91f3-b7b21d1a1f53)

**Reply by paolomariaschiratti** [reply]
Issues fixed moving to a different PC. It was probably related to my company security restrictions
